src/main/java/org/littil/api/feedback/api/FeedbackResource.java

```diff
@@ -0,0 +1,44 @@
+package org.littil.api.feedback.api;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.littil.api.mail.MailService;
+
+import java.util.Optional;
+
+@Path("/api/v1/feedback")
+@RequestScoped
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "Feedback")
+public class FeedbackResource {
+
+    @Inject
+    MailService mailService;
+    @Inject
```

Comment:
should this property be injected into MailService?

Reply:
property moved to MailService